docs/DeveloperGuide.md

```diff
@@ -260,6 +261,46 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Set/Unset payment made feature
+
+#### Implementation
+
+The purpose of the set/unset payment made feature is for tutors to modify a student's payment status.
+
+This feature implements the following operations:
+
+* `PaidCommand#execute()` - Updates the payment status of the student to paid.
+* `UnpaidCommand#execute()` - Updates the payment status of the student to unpaid.
+
+This feature is facilitated by the following operations:
+
+* `TutorAidParser#parseCommand()` - Calls `PaidCommandParser#parse()` or `UnpaidCommandParser#parse()` with the specified student index, depending on the command word in the user input.
+* `PaidCommandParser#parse()` - Returns an instance of `PaidCommand`.
+* `UnpaidCommandParser#parse()` - Returns an instance of `UnpaidCommand`.
+
+To represent a student's payment status, a `PaymentStatus` class is created. It stores an immutable instance variable `hasPaid`, of boolean type. We then work with the `Student` model, and implement it as a field in Student.
+
+<img src="images/StudentWithPaymentStatusClassDiagram.png" width="150" />
+
+Given below is an example usage scenario for setting a student's payment status as paid, and how the command is executed.
+
+Step 1. The user executes `paid 2` command to set the payment status of the 2nd student in the address book. `LogicManager#execute()` is executed, where the user input is passed into `TutorAidParser#parseCommand()`. This in turn calls `PaidCommandParser#parse()`, which then checks for the validity of the provided index. If the index is valid, `PaidCommandParser#parse()` returns a `PaidCommand` instance.
```

Comment:
Can we just use the markdown style numbering here, e.g.
1. something
2. something
3. something

instead of Step 1. Step 2. etc so that lines get indented

Also we can consider removing the "which then checks for the validity of the provided index" as it is inferred from the next sentence.

Reply:
Changed to markdown style numbering, and has removed the specified statement too. Thanks!